src/libraries/System.Net.NameResolution/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -63,7 +63,6 @@ public void GetHostEntry_InvalidHost_LogsError()
         }
 
         [ConditionalFact]
-        [PlatformSpecific(~TestPlatforms.Windows)]  // Unreliable on Windows.
```

Comment:
I'm skeptical any time I see a construct like:
```C#
await Task.Delay(100).ConfigureAwait(false);
```
used for synchronization.  But if it's not been failing on Unix, I'm not sure why it'd be any more likely to fail on Windows.

Reply:
Yep, that's not ideal. Is for the event source a better way to "wait" until the log is written?

Reply:
I'm wondering if we could emit some other known event and wait for that to arrive. And assume the events are processed in sequence. @MihaZupan  would probably know how the events are processed.

Reply:
We should use something like [`WaitForEventAsync`](https://github.com/dotnet/runtime/blob/a3b37a289b00c748eb92bd4eb4dc78146bd72176/src/libraries/System.Net.Sockets/tests/FunctionalTests/TelemetryTest.cs#L418) we used in Sockets telemetry across all such tests.